FastSimulation/Tracking/plugins/TrajectorySeedProducer.cc

```diff
@@ -164,69 +159,90 @@ TrajectorySeedProducer::beginRun(edm::Run const&, const edm::EventSetup & es)
     magneticFieldMap = &(*magneticFieldMapHandle);
     trackerTopology = &(*trackerTopologyHandle);
 
-    thePropagator = new PropagatorWithMaterial(alongMomentum,0.105,magneticField); 
+    thePropagator = std::move(std::shared_ptr<PropagatorWithMaterial>(new PropagatorWithMaterial(alongMomentum,0.105,magneticField)));
```

Comment:
There is no need to call ```std::move``` here since it is already an 'R value reference' (i.e. an unnamed variable). In fact, it would be best to do
```
thePropagator = std::make_shared<PropagatorWithMaterial>(alongMomentum,0.105,magneticField);
```
this does 1 memory allocation instead of 2 when creating the ```std::shared_ptr```.